shiny_tutorial.Rmd

```diff
@@ -0,0 +1,20 @@
+
+```{r  include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
```

Comment:
This part should not exist before the header.

Reply:
Problem is fixed now. Do I need to submit another pull request? Thank you!

Reply:
No, you don't need to. Thank you!